dbt_semantic_interfaces/protocols/__init__.py

```diff
@@ -27,6 +27,7 @@
 )
 from dbt_semantic_interfaces.protocols.semantic_model import (  # noqa:F401
     SemanticModel,
+    SemanticModelConfig,
```

Comment:
Is there a reason we also didn't add `MetricConfig` to the imports from `metric` on line 13?